docs/getting-started/advanced-setting.asciidoc

```diff
@@ -64,7 +56,7 @@ data shipped via {beats} and the {agent} is automatically added to the
 
 You can add or remove any indices and index patterns as required. For background information on {es} indices, refer to {ref}/documents-indices.html[Data in: documents and indices].
 
-NOTE: If you leave the `logs-*` index selected, by default, all Elastic cloud logs are excluded from all queries in the {security-app}. This is to avoid adding data from cloud monitoring to the app.
+NOTE: If you leave the `-*elastic-cloud-logs-*` index pattern selected, by default, all Elastic cloud logs are excluded from all queries in the {security-app}. This is to avoid adding data from cloud monitoring to the app.
```

Comment:
Optional edit to improve flow.

```suggestion
NOTE: If you leave the `-*elastic-cloud-logs-*` index pattern selected, all Elastic cloud logs are excluded from all queries in the {security-app} by default. This is to avoid adding data from cloud monitoring to the app.
```